### PR TITLE
Updated .classpath files

### PR DESCRIPTION
**Pull request by szarnekow**

### CODE REVIEW

File: org.eclipse.xtext.builder/.classpath [hunk 1]
```diff
@@ -2,13 +2,9 @@
...
-	<classpathentry kind="src" path="xtend-gen"/>
-	<classpathentry kind="src" path="emf-gen">
-		<attributes>
-			<attribute name="ignore_optional_problems" value="true"/>
-		</attributes>
-	</classpathentry>
-	<classpathentry kind="src" path="deprecated"/>
-	<classpathentry kind="output" path="bin"/>
+	<classpathentry kind="src" path="emf-gen/"/>
+	<classpathentry kind="src" path="src/"/>
```
Comment (kthoms) [inline org.eclipse.xtext.builder/.classpath]: why emf-gen before src?

Comment (szarnekow) [reply org.eclipse.xtext.builder/.classpath]: Seems to depend on the order in the build.properties (which does make sense). I'll reorder it there.

Comment (kthoms) [reply org.eclipse.xtext.builder/.classpath]: FYI The order can be defined in the build path and affects the order in which the containers are listed in the navigator tree.

Comment (szarnekow) [reply org.eclipse.xtext.builder/.classpath]: See next commit

**Pull request merged**